server/command_test.go

```diff
@@ -238,6 +239,132 @@ func TestExecuteShowCommand(t *testing.T) {
 	}
 }
 
+func TestExecuteShowLinksCommand(t *testing.T) {
+	p := newTestPlugin(t)
+	mockAPI := &plugintest.API{}
+
+	for _, testCase := range []struct {
+		description string
+		args        *model.CommandArgs
+		setupAPI    func(*plugintest.API)
+		setupStore  func(*mockStore.Store)
+		setupClient func(*mockClient.Client)
+	}{
+		{
+			description: "Successfully executed show-links command",
+			args: &model.CommandArgs{
+				UserId:    testutils.GetUserID(),
+				ChannelId: testutils.GetChannelID(),
+			},
+			setupAPI: func(api *plugintest.API) {
+				api.On("GetTeam", testutils.GetTeamID()).Return(&model.Team{DisplayName: "Test MM team"}, nil).Times(1)
+				api.On("GetChannel", testutils.GetChannelID()).Return(&model.Channel{DisplayName: "Test MM channel"}, nil).Times(1)
+
+				api.On("SendEphemeralPost", testutils.GetUserID(), &model.Post{
+					UserId:    "bot-user-id",
+					ChannelId: testutils.GetChannelID(),
+					Message:   "Please wait while your request is being processed.",
+				}).Return(testutils.GetPost(testutils.GetChannelID(), testutils.GetUserID())).Once()
+
+				api.On("SendEphemeralPost", testutils.GetUserID(), &model.Post{
+					UserId:    "bot-user-id",
+					ChannelId: testutils.GetChannelID(),
+					Message:   "| Mattermost Team | Mattermost Channel | MS Teams Team | MS Teams Channel | \n| :------|:--------|:-------|:-----------|\n|Test MM team|Test MM channel|Test MS team|Test MS channel|",
```

Comment:
can create a constant for this table header layout so that same can be used at all such places also if in the future if we need to modify anything will need to change at only one place

Reply:
I don't think there's any need to make this constant as this will not be used anywhere else except the unit tests.

Reply:
I mean creating this part as constant `"| Mattermost Team | Mattermost Channel | MS Teams Team | MS Teams Channel | \n| :------|:--------|:-------|:-----------|\n"` so that we can concatenate this to the rest of table content wherever needed like test cases and the ephemeral post

Reply:
I understood you. This string will be used only in this function and only once, that's why I don't think there's a need to make this constant. For unit tests also, I don't think we should use a constant. The reason is whenever someone changes this string, the unit tests will fail and I want them to fail so that we can be aware of the changes. If we use a constant, then this will not be possible.